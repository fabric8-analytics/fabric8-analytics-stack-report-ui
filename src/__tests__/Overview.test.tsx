import React from "react";
import { shallow, configure } from "enzyme";
import OverviewCard from "../app/components/main-components/Overview/Overview";

describe("<App />", () => {
  test("Should render the app Layout", async () => {
    const component = shallow(<OverviewCard />);
    expect(component).toMatchSnapshot();
  });
});
