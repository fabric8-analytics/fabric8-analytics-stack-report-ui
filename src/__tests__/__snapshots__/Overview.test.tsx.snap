// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Should render the app Layout 1`] = `
<Card
  className="GridCard"
  isFullHeight={true}
  isHoverable={true}
>
  <CardTitle>
    <TextContent>
      <Text
        className="overview-title"
      >
        Overview of the Stack
      </Text>
    </TextContent>
  </CardTitle>
  <CardBody
    className="CardBodySize"
  >
    <Tabs
      activeKey={0}
      component="div"
      isBox={false}
      isFilled={false}
      isSecondary={false}
      isVertical={false}
      leftScrollAriaLabel="Scroll left"
      mountOnEnter={false}
      onSelect={[Function]}
      ouiaSafe={true}
      rightScrollAriaLabel="Scroll right"
      unmountOnExit={false}
      variant="default"
    >
      <Tab
        eventKey={0}
        onSelect={[Function]}
        title={
          <React.Fragment>
            <TabTitleIcon>
              <SecurityIcon
                color="currentColor"
                noVerticalAlign={false}
                size="sm"
              />
            </TabTitleIcon>
            <TabTitleText>
              Security Issues
            </TabTitleText>
          </React.Fragment>
        }
      >
        <br />
        <OverviewContent />
      </Tab>
      <Tab
        eventKey={1}
        title={
          <React.Fragment>
            <TabTitleIcon>
              <ZoneIcon
                color="currentColor"
                noVerticalAlign={false}
                size="sm"
              />
            </TabTitleIcon>
            <TabTitleText>
              Add-ons
            </TabTitleText>
          </React.Fragment>
        }
      >
        <DrawerFC />
      </Tab>
    </Tabs>
  </CardBody>
</Card>
`;
